services/Skyra.Database/Models/Entities/Guild.cs

```diff
@@ -743,6 +743,14 @@ public class Guild
 		[Column("notifications.streams.twitch.streamers", TypeName = "jsonb")]
 		public string NotificationsStreamsTwitchStreamers { get; set; } = null!;
 
+		[Required]
+		[Column("notifications.youtube.channel", TypeName = "character varying(19)")]
+		public string YoutubeNotificationChannel { get; set; } = null!;
+
+		[Required]
+		[Column("notifications.youtube.message")]
+		public string YoutubeNotificationMessage { get; set; } = null!;
```

Comment:
Those can be nullable:
```suggestion
		[Column("notifications.youtube.channel", TypeName = "character varying(19)")]
		public ?string YoutubeNotificationChannel { get; set; };

		[Column("notifications.youtube.message")]
		public ?string YoutubeNotificationMessage { get; set; };
```